README.md

```diff
@@ -1 +1,59 @@
-# leader-board
\ No newline at end of file
+# Leaderboard
+
+
+In this project, I built a simple HTML Leaderboard . I built it using HTML, CSS, JavaScript,EcmaScript6 modules webpack and served it by a webpack dev server.
+
+
+Template for Leaderboard setup
+
+🗼Built With
+HTML
+JavaScript
+css
+EcmaScript6
+modules
+GitHub Actions
+Webpack
+✈️ Launch
+Getting Started
+⚠️⚠️ Note the commands for running the server⚠️⚠️
+npm run start
+
+Setting up this project locally
+How should it be tested
+Clone the repository
+
+ git clone https://github.com/Ibtisam34/leader-board.git
+Navigate to the folder
+
+🎉Congratulations you're done✨
+
+
+## Authors
+
+👤 Riyana
+
+- GitHub: [ibtisam34](https://github.com/ibtisam34)
+- LinkedIn: [LinkedIn](linkedin.com/in/riyangwl2)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+ 
+  ## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
### OPTIONAL

- Kindly use the updated version  of the microverse [readme](https://github.com/microverseinc/readme-template) template